h/views/api/config.py

```diff
@@ -108,10 +108,10 @@ def api_config(versions, link_name=None, description=None, **settings):
     `add_api_view` to register the view instead of `context.add_view`.
     """
 
-    def callback(context, _name, ob):
+    def callback(context, _name, view_ob):
         add_api_view(
             context.config,
-            view=ob,
+            view=view_ob,
```

Comment:
What's an "ob" even if it's a "view_ob". Observer or object or something?

Reply:
yeah, why not just `view`?